src/redux/modules/mdb.js

```diff
@@ -76,21 +78,25 @@ export const actions = {
 
   receiveCollections,
   receiveContentUnits,
+  receiveSources,
```

Comment:
Anything related to sources data should live in the 'sources' redux module.
'mdb' redux module is for collections and units.

Please move all this things to sources module and try to reuse code as much as possible.

Reply:
Fixed.